beacon-chain/core/state/BUILD.bazel

```diff
@@ -12,11 +12,12 @@ go_library(
     importpath = "github.com/prysmaticlabs/prysm/beacon-chain/core/state",
     visibility = [
         "//beacon-chain:__subpackages__",
+        "//endtoend:__pkg__",
         "//shared/interop:__pkg__",
         "//shared/testutil:__pkg__",
         "//tools/benchmark-files-gen:__pkg__",
         "//tools/genesis-state-gen:__pkg__",
-        "//endtoend:__pkg__",
```

Comment:
is there a reason to replace end to end?

Reply:
not replaced its just shifted above ^

Reply:
missed it. sorry